### ISSUE TITLE
NullPointerException

### ISSUE THREAD

**Issue by athese6** [bumptech/glide#572]
**Glide Version/Integration library (if any)**:
glide-4.0.0-SNAPSHOT

**Device/Android Version**:
LG-F320S / 4.2.2

**Issue details/Repro steps**:

**Glide load line**:
```
Glide.with(context)...;
```

**Stack trace**:
```
paste stack trace here
E/AndroidRuntime﹕ FATAL EXCEPTION: main
    java.lang.NullPointerException
            at com.bumptech.glide.load.engine.GlideException$IndentedAppendable.append(GlideException.java:202)
            at com.bumptech.glide.load.engine.GlideException.appendExceptionMessage(GlideException.java:142)
            at com.bumptech.glide.load.engine.GlideException.appendCausesWrapped(GlideException.java:175)
            at com.bumptech.glide.load.engine.GlideException.appendCauses(GlideException.java:153)
            at com.bumptech.glide.load.engine.GlideException.printStackTrace(GlideException.java:126)
            at com.bumptech.glide.load.engine.GlideException.appendCausesWrapped(GlideException.java:173)
            at com.bumptech.glide.load.engine.GlideException.appendCauses(GlideException.java:153)
            at com.bumptech.glide.load.engine.GlideException.printStackTrace(GlideException.java:126)
            at com.bumptech.glide.load.engine.GlideException.appendCausesWrapped(GlideException.java:173)
            at com.bumptech.glide.load.engine.GlideException.appendCauses(GlideException.java:153)
            at com.bumptech.glide.load.engine.GlideException.printStackTrace(GlideException.java:126)
            at com.bumptech.glide.load.engine.GlideException.printStackTrace(GlideException.java:121)
            at android.util.Log.getStackTraceString(Log.java:319)
            at android.util.Log.w(Log.java:191)
            at com.bumptech.glide.request.SingleRequest.onLoadFailed(SingleRequest.java:488)
            at com.bumptech.glide.load.engine.EngineJob.handleExceptionOnMainThread(EngineJob.java:250)
            at com.bumptech.glide.load.engine.EngineJob.access$200(EngineJob.java:23)
            at com.bumptech.glide.load.engine.EngineJob$MainThreadCallback.handleMessage(EngineJob.java:279)
            at android.os.Handler.dispatchMessage(Handler.java:95)
            at android.os.Looper.loop(Looper.java:137)
            at android.app.ActivityThread.main(ActivityThread.java:5168)
            at java.lang.reflect.Method.invokeNative(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:511)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:797)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:564)
            at dalvik.system.NativeStart.main(Native Method)

**Comment by TWiStErRob** [comment]
What is your Glide load line?

**Comment by athese6** [comment]
requestBuilder = requestManager.asDrawable().apply(RequestOptions.fitCenterTransform(context)).apply(diskCacheStrategyOf(DiskCacheStrategy.ALL));

requestBuilder.clone().apply(RequestOptions.signatureOf(new StringSignature(...))).load(...).thumbnail(0.25f).into(...);

**Comment by sjudd** [comment]
Internal bug is b/23486085

**Comment by specialren** [comment]
has fixed? how to avoid?

**Comment by sjudd** [comment]
This is fixed here: a64d5d0af38facfe9cf5b8ff0efb7ac6b4aebb4b
